addon/components/nav-stack.js

```diff
@@ -415,6 +427,9 @@ export default class NavStack extends Component {
   cloneHeader() {
     this.removeClonedHeader();
     let liveHeader = this.element.querySelector('.NavStack-currentHeaderContainer');
+    if (!liveHeader) {
```

Comment:
When would there not be a life header? Maybe there should be a comment here

Reply:
I'm not recalling when this could happen. Perhaps on initial render?